en_US/dashboard/audit-log.md

```diff
@@ -1,32 +1,32 @@
 
-Audit logs for command-line operations may contain sensitive information, so take caution when sending them to a log collector. It is advisable to filter log content or use encrypted transmission methods to prevent unauthorized information leakage.
+EMQX offers a Dashboard view and integration with log systems to help enterprises manage audit logs. Through these methods EMQX provides flexible and comprehensive support for Audit Logs, allowing enterprise users to choose the most suitable way to manage and view audit logs according to their needs.
 
-:::
+## Enable Audit Log
 
-## Dashboard
+You can enable the Audit Log feature and adjust the configuration parameters through both the Dashboard and configuration file.
 
-<!-- TODO 5.4 -->
-
-![image-20231214143911786](./assets/image-20231214143911786.png)
+### Enable Audit Log via Dashboard
 
-## View Log File
+You can enable the audit log and modify the configuration parameters in the Dashboard by navigating to **Management** -> **Logging** -> **Audit Log**.
 
-When the Audit Log is enabled in EMQX, change-related operations are saved in log format in the `./log/audit.log.1` file. It is easy for enterprise users to perform an in-depth analysis of audit records and integrate the results into existing log management systems, meeting compliance and data security requirements.
-
-In future versions, EMQX will also support direct viewing in the Dashboard or querying and obtaining Audit Logs via the REST API, making it convenient to check operation records. Through these methods, EMQX provides flexible and comprehensive support for Audit Logs, allowing enterprise users to choose the most suitable way to manage and view audit logs according to their needs.
+<img src="./assets/audit_log_config.png" alt="Audit Log Configuration" style="zoom:50%;" />
 
-## Configure Audit Log
+Audit log configuration options:
 
-You can enable the Audit Log feature and adjust the configuration parameters through both the configuration file and the Dashboard.
+- **Enable Log Handler**: Toggle the switch to enable or disable the audit log processing process. It is enabled by default.
+- **Audit Log File Name**: Specify the path and name of the audit log file. The default value is `${EMQX_LOG_DIR}/audit.log`, where `${EMQX_LOG_DIR}` is a variable and defaults to `./log`, meaning it is ultimately be saved in `./log/audit.log.1`.
+- **Maximum Log Files Number**: The maximum number of rotated log files. The default value is `10`.
+- **Rotation Size**: Set the size of log files, and when the specified size is reached, log files will be rotated. If disabled, log files will grow indefinitely. You can enter the desired value in the text box and select the unit from the dropdown list, with options such as `MB`, `GB`, and `KB`. The default value is `50MB`.
```

Comment:
Missing 2 configuration items
- Max Filter Size
- Ignore High Frequency Request

Reply:
Do we also need to update the configuration example:

log.audit {
  enable = true
  path = "./log/audit.log"
  rotation_count = 10
  rotation_size = 50MB
  time_offset = system
}